client/src/main/java/org/eclipse/hono/client/impl/HonoClientImpl.java

```diff
@@ -757,6 +758,91 @@ private void getOrCreateSender(
         });
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public final Future<MessageConsumer> createCommandResponseConsumer(
+            final String tenantId,
+            final String replyId,
+            final Consumer<Message> createCommandResponseConsumer,
+            final Handler<Void> closeHandler) {
+
+        return createCommandResponseConsumer(tenantId, replyId,
+                (delivery, message) -> createCommandResponseConsumer.accept(message), closeHandler);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public final Future<MessageConsumer> createCommandResponseConsumer(
+            final String tenantId, final String replyId,
+            final BiConsumer<ProtonDelivery, Message> messageConsumer,
+            final Handler<Void> closeHandler) {
+
+        return createConsumer(
+                tenantId,
+                () -> newCommandResponseConsumer(tenantId, replyId, messageConsumer, closeHandler));
+    }
+
+    private Future<MessageConsumer> newCommandResponseConsumer(
```

Comment:
do we still need this?

Reply:
ups, so sorry. Something went terribly wrong with my local merging.